db/Categorie.js

```diff
@@ -0,0 +1,24 @@
+import { DataTypes, Model } from 'sequelize';
+import { sequelize } from './sequelize.js';
+
+export class Category extends Model {}
```

Comment:
We currently use two different styles for defining models.
Maybe we can bring this up during daily and agree on a single approach for consistency?
@Oleksandra-Mu